.github/workflows/ci.yml

```
@@ -0,0 +1,43 @@
+name: Build the documentation
+
+on:
+  pull_request:
```

This should also trigger on pushes:

```suggestion
on:
  push:
  pull_request:
```